src/components/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  todos: Todo[];
+  filter: 'all' | 'active' | 'completed';
+  setFilter: (filter: 'all' | 'active' | 'completed') => void;
+  clearCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filter,
+  setFilter,
+  clearCompleted,
+}) => {
+  const leftItems = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {leftItems} items left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: filter === 'all',
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => setFilter('all')}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: filter === 'active',
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => setFilter('active')}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={classNames('filter__link', {
+            selected: filter === 'completed',
+          })}
+          data-cy="FilterLinkCompleted"
+          onClick={() => setFilter('completed')}
+        >
+          Completed
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method